docs/proposals/vertical-pod-autoscaler.md

```diff
@@ -0,0 +1,75 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/proposals/vertical-pod-autoscaler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Abstract
+
+Vertical Pod Autoscaler is a feature that periodically adjust pods' [Resources](resource-qos.md#resource-specifications) based on demand signals.
+This is a natural extension of [Initial Resources](initial-resources.md) plugin.
+
+**Note:** since there is no plan to implement the feature immediately (as for 9/9/15) this document is just an overview how it should be done
+rather than detailed design.
+
+## Motivation
+
+Having Resources set to a correct value is very important: if the value is too high resources in the cluster are wasted,
```

Comment:
The exact effect of setting {memory, CPU} {request, limit} too {high, low} is actually quite complicated due to over-commit, CPU shares, CPU quota, OOM handling, and cgroup memory containment. You could analyze all 8 cases but I think it would be better to just make your statement even more vague than it already is -- saying something like "Setting the value incorrectly could lead to wasted resources, violating service latency objectives, or one or more containers being killed because the machine ran out of memory."